tests/unit/Generator/GenerateFunctionTest.php

```diff
@@ -126,6 +129,7 @@ public function testWithReflectionException()
     {
         $this->expectException(InvalidRouteException::class);
         $this->expectExceptionMessage("Invalid route for 'GET /*'. Can't call info()");
+        $this->expectExceptionCode(0);
```

Comment:
why it's bad. Don't understand. Strict checks

Reply:
I don't explicitly set the exception code. `0` is just the default of the `Exception` class. If the default was anything else, it would return that. If we choose a different base class, it will return that. The code is an unknown / [don't care](https://en.wikipedia.org/wiki/Don%27t-care_term) and thus shouldn't be checked.